Runtime/Helpers/EnumHelper.cs

```diff
@@ -46,5 +46,52 @@ T max_inclusive
 
             return ( T )array.GetValue( UnityEngine.Random.Range( min_index, max_index + 1 ) );
         }
+
+        public static uint GetToggledFlagsCount<T>(
+            T flags
+            ) where T : Enum
+        {
+            long flags_as_long = Convert.ToInt64( flags );
+
+            if( flags_as_long == -1 )
+            {
+                return ( uint )GetValueTable<T>().Count() - 1;
+            }
```

Comment:
This fails to work as the user would expect if there is for example such an enum:
```cs
A = 0,
B = 1,
C = 2,
Count = 3,

Everything = ~0,
```

Reply:
My bad, this is for flags. It still fails in some cases regardless, take this enum:
```cs
None = 0,
A = 1 << 0,
B = 1 << 1,
C = 2 << 2,
D = 2 << 3,

ABC = A | B | C,

Everything: ~0,
```
Take `~ABC`, it would say there's 61 flags.

Reply:
Right. I moved that method as an ulong extension then so the user cannot have wrong assumptions anymore.